src/lib/transaction_snark/test/util.ml

```diff
@@ -7,16 +7,35 @@ module Impl = Pickles.Impls.Step
 module Zkapp_command_segment = Transaction_snark.Zkapp_command_segment
 module Statement = Transaction_snark.Statement
 
-let constraint_constants = Genesis_constants.Compiled.constraint_constants
-
 let genesis_constants = Genesis_constants.Compiled.genesis_constants
 
 (* Always run tests with proof-level Full *)
 let proof_level = Genesis_constants.Proof_level.Full
 
+let constraint_constants : Genesis_constants.Constraint_constants.t =
+  { sub_windows_per_window = 1
+  ; ledger_depth = 35
+  ; work_delay = 1
+  ; block_window_duration_ms = 1
+  ; transaction_capacity_log_2 = 1
+  ; pending_coinbase_depth = 1
+  ; coinbase_amount = Currency.Amount.zero
+  ; supercharged_coinbase_factor = 1
+  ; account_creation_fee = Currency.Fee.of_mina_string_exn "0.1"
+  ; fork = None
+  }
+
+let protocol_constants : Genesis_constants.Protocol.t =
+  { k = 1
+  ; slots_per_epoch = 1000
+  ; slots_per_sub_window = 1
+  ; grace_period_slots = 1
+  ; delta = 1
+  ; genesis_state_timestamp = Int64.one
+  }
+
 let consensus_constants =
-  Consensus.Constants.create ~constraint_constants
-    ~protocol_constants:genesis_constants.protocol
+  Consensus.Constants.create ~constraint_constants ~protocol_constants
```

Comment:
Are you using this file? Why are you changing it?

Reply:
Not directly, the dependency is `all zeko clis -> cli_lib -> snark_profiler_lib -> transaction_snark_tests`

Reply:
Added comment